source/partials/schedule/_program.slim

```diff
@@ -0,0 +1,14 @@
+.Program
```

Comment:
I was thinking more along the lines of "daily program". But now that I think of it, it also makes sense to say "daily schedule". Any suggestions?